mmv1/products/servicemanagement/api.yaml

```diff
@@ -36,3 +36,18 @@ objects:
         name: 'serviceName'
         description: The name of the service.
         required: true
+  - !ruby/object:Api::Resource
+    name: 'ServiceConsumers'
+    base_url: services/{{service_name}}/consumers/{{consumer_project}}
+    self_link: services/{{service_name}}/consumers/{{consumer_project}}
+    description: |
+      A service that is managed by Google Service Management
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'serviceName'
+        description: The name of the service.
+        required: true
+      - !ruby/object:Api::Type::String
+        name: 'consumerProject'
+        description: Service consumer level permissions require that the request to the Service Control API must contain at least one valid service consumer project ID.
```

Comment:
Can we improve the description here? Is this the project that IAM is granting permission to or for, or how would a user decide what the consumer project is?

Reply:
Starting discussion with the team to try and understand this field with more clarity.